tests/files/opds/content_server_mini.opds

```diff
@@ -63,5 +63,4 @@
    <schema:description>I'm working to locate a source for this identifier.</schema:description>
   </simplified:message>
 
-  <link href="http://localhost:5000/?after=327&amp;size=100" type="application/atom+xml;profile=opds-catalog;kind=acquisition" rel="next"/>
```

Comment:
The way this test file was used in the old tests, it didn't matter that it contained a next link. However in the new tests, if the feed has a next link, we try to follow it, since we are testing the whole importer chain. Instead of working around this, I removed the next link from this feed.

I left the next links in a couple of the feeds, and also test that case.